### PR TITLE
Sunwell node

### PR DESCRIPTION
**Pull request by martinbonnin**
type `node generate_images.js enUS frFR koKR ...` to generate all the PNGs

requires a bunch of dependencies, artwork and font files. One key dependency is very updated node-canvas to have support for font loading. 

I intentionally tried to change as little code as possible but the code could certainly be refactored a bit.

### CODE REVIEW

File: generate_images.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+var Canvas = require('canvas')
+  , Image = Canvas.Image;
+var fs = require("fs");
+var path = require('path')
+
+console.log("hello");
```
Comment (jleclanche) [inline generate_images.js]: hello!

Comment (jleclanche) [reply generate_images.js]: This whole file contains a lot of unnecessary logging/debug stuff. Did you mean to commit it?

File: generate_images.js [hunk 2]
```diff
@@ -71,20 +76,75 @@ var leeroy = {
...
+			if (!error && response.statusCode === 200) {
+				const j = JSON.parse(body)				
+				drawAllCards(j, dir);
+			} else {
+				console.log("Got an error: ", error, ", status code: ", response.statusCode)
+			}
+		})
+	}
+	mkdirp(dir, function(err) {
+		downloadJson("https://api.hearthstonejson.com/v1/15590/" + lang + "/cards.json");
```
Comment (jleclanche) [inline generate_images.js]: Can you make this a global/configurable somehow? You can also use `latest` instead of  a build to always get the latest one.

**Review by jleclanche — CHANGES_REQUESTED** [review]
Looks great. Couple changes to squash and this can be merged.

**Comment by andburn** [comment]
You could maybe add a small piece in the Readme about it, dependencies and such. This is what I needed anyway:
```
sudo apt-get install libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev build-essential g++
npm install -g node-gyp
npm install github:automattic/node-canvas
npm install mkdirp
npm install promise
npm install request
```

File: generate_images.js (hunk 1 above)
Comment (martinbonnin) [reply generate_images.js]: I just removed some of it. I still it find interesting to have the current progress of the operation though so I kept the 'save x/2819 XX.png'

File: generate_images.js (hunk 2 above)
Comment (martinbonnin) [reply generate_images.js]: done

**Comment by martinbonnin** [comment]
Good idea for the readme, I just added a papragraph. Else, I use this bootstrap script to setup my dev environment: https://gist.github.com/martinbonnin/af2df4d1f1eadaa05645a22512556413. Maybe we could version control that somewhere

**Comment by jleclanche** [comment]
Landed in a328138acfc0ec5bbb2e165f319cf0107b5a6685

**Pull request closed without merging**